README.md

```diff
@@ -1,330 +1,259 @@
+3. Track your Hamilton DAG by create a `HamiltonTracker` object with your `username` and `project_id`. Now, all executions will be tracked and available from the UI!
 
-We also have a docker container that contains some of these examples so you can pull that and run them locally. See the [examples folder README](https://github.com/DAGWorks-Inc/hamilton/blob/main/examples/README.md#running-examples-through-a-docker-image) for details.
+   ```python
+   from hamilton import driver
+   from hamilton_sdk.adapters import HamiltonTracker
+   import my_dag
+
+   # use your `username` and `project_id`
+   tracker = HamiltonTracker(
+      username="my_username",
+      project_id=1,
+      dag_name="hello_world",
+   )
+
+   # building the `Driver` will add the DAG to the catalog
+   dr = (
+      driver.Builder()
+      .with_modules(my_dag)
+      .with_adapters(tracker)  # add your tracker here
+      .build()
+   )
+
+   # executing the `Driver` will track results
+   dr.execute(["C"])
+   ```
 
-# We forked and lost some stars
-This repository is maintained by the original creators of Hamilton, who have since founded [DAGWorks inc.](https://dagworks.io/), a company largely dedicated to building and maintaining the Hamilton library. We decided to fork the original because Stitch Fix did not want to transfer ownership to us; we had grown the star count in the original repository to 893: <img width="141" alt="Screen Shot 2023-02-23 at 12 58 43 PM" src="https://user-images.githubusercontent.com/2328071/221029806-36a4e088-809b-4b08-9199-2fbf8049be34.png">
-before forking.
+# Documentation & learning resources
+
+* 📚 See the [official documentation](https://hamilton.dagworks.io/) to learn about the core concepts of Hamilton.
+
+* 👨‍🏫 Consult the [examples on GitHub](https://github.com/DAGWorks-Inc/hamilton/tree/main/examples) to learn about specific features or integrations with other frameworks.
+
+* 📰 The [DAGWorks blog](https://blog.dagworks.io/) includes narrative tutorials and broader guides about how to build a data platform.
+
+* 📺 Find video tutorials on the [DAGWorks YouTube channel](https://www.youtube.com/@DAGWorks-Inc)
+
+* 📣 Reach out via the [Hamilton Slack community](https://join.slack.com/t/hamilton-opensource/shared_invite/zt-1bjs72asx-wcUTgH7q7QX1igiQ5bbdcg) for help and troubleshooting
+
+
+# How does Hamilton compare to X?
+
+Hamilton is not an orchestrator ([you might not need one](https://blog.dagworks.io/p/lean-data-automation-a-principal)) nor a feature store ([but you can use it to build one!](https://blog.dagworks.io/p/featurization-integrating-hamilton)). It's closest to dbt; it helps you structure and manage data transformations. While dbt is focus on SQL and data tables, Hamilton supports the full Python ecosystem and is great for machine learning pipelines, working with documents & LLMs, web requests, and more.
```

Comment:
OK, we don't want to lose people with assuming dbt knowledge. "If you know dbt -- hamilton does for python what DBT does for sql -- taking simliar (python-specific) standards and applying them to ..."